samples/nrf9160/mqtt_simple/src/main.c

```diff
@@ -449,6 +449,12 @@ static void button_handler(uint32_t button_states, uint32_t has_changed)
  */
 static int modem_configure(void)
 {
+
+	/* Turn off power saving modes for a more responsive demo */
```

Comment:
perhaps update this comment to indicate that it should be done before net reg?

Reply:
Yes. Lets add that.